cmds/deploy.go

```diff
@@ -217,6 +223,7 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 		},
 	}
 	cmd.PersistentFlags().StringP("domain", "d", defaultDomain(), "The domain name to append to the service name to access web applications")
+	cmd.PersistentFlags().StringP("apidomain", "", "--domain", "if set the fabric8 pod uses that domain to call the console")
```

Comment:
@rawlingsj , @jimmidyson yeah I can address these issues. But not quite sure what exactly is meant. you mean the new param should be api-server? It is passed to the template as $DOMAIN where it is used as OAUTH_AUTHORIZE_URI and GOOGLE_OAUTH_REDIRECT_URI. Not sure if api-server is misleading.
And providing defaultDomain() as default value is not correct IMHO. Cause if it isn't set the value of the domain param is used.  But the comment is wrong.. it isn't the console being called but the rest api.